src/web/src/components/MainPhotos/PhotosGrid/index.vue

```diff
@@ -0,0 +1,165 @@
+              :key="i"
+              link
+              
+            >
+              <v-list-item-title>{{ item.name }}</v-list-item-title>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+      </v-menu>
+      <v-spacer></v-spacer>
+      <Search :data="photos"/>
+      <v-btn color="primary" @click="handleClick('23')">
+         <v-icon>mdi-plus</v-icon>
+        <div>
+          Add Photo
+        </div>
+      </v-btn>
+    </v-app-bar>
+  <v-container grid-list-xs>
+    <v-text-field v-model="search" label="Search"></v-text-field>
+    <v-row>
+      <v-col
+        v-for="(item, i) in sortedData"
+        :key="`photo-${i}`"
+        class="d-flex child-flex"
+        cols="2"
+      >
+        <v-hover>
+          <template v-slot:default="{ hover }">
+            <v-card
+              class="mx-auto"
+              @click="sortData()"
+            >
+              <v-img
+                :src="item.photo"
+                :lazy-src="item.photo"
+                class="white--text align-end"
+                gradient="to bottom, rgba(0,0,0,.1), rgba(0,0,0,.5)"
+              >
+              </v-img>
+
+              <v-card-actions>
+                <v-card-subtitle v-if="selectedSorter == 0" v-text="item.name"></v-card-subtitle>
+                <v-card-subtitle v-else-if="selectedSorter == 1" v-text="`Rating: ${item.rating}`"></v-card-subtitle>
+                <v-card-subtitle v-else v-text="`Photo taken, ${new Date(item.date).toLocaleDateString()}`"></v-card-subtitle>
+              </v-card-actions>
+
+              <v-fade-transition>
+                <v-overlay
+                  v-if="hover"
+                  absolute
+                  color="#036358"
+                >
+                  <v-btn @click="handleClick(i)">Edit Photo</v-btn>
+                </v-overlay>
+              </v-fade-transition>
+            </v-card>
+          </template>
+        </v-hover>
+      </v-col>
+    </v-row>
+  </v-container>
+    
+  
+
+  </div>
+</template>
+
+<script>
+import Search from "../PhotosComponents/Search";
+export default {
+  name: "Grid",
+  components: { Search },
+  data: () => ({
+    search: "",
+    selectedSorter: 0,
+    sortOptions: [
+      {name: 'Feature name'},
+      {name: 'Rating'},
+      {name: 'Age'},
+    ],
+    photos: [],
+    sortedData: []
+
+  }),
+  watch: {
+    search: {
+      handler() {
+        //this.getDataFromApi();
+        this.sortData();
+      },
+      deep: true,
+    },
+  },
+  mounted() {
+    this.getDataFromApi();
+  },
+  methods: {
+    handleClick(value){   //Redirects the user to the site form
+        this.$router.push(`/photos/${value}/feature`);
+    },
+    getDataFromApi() {
+      for(let i =0; i<12; i++){
+        this.photos.push({id: i, name: `photo-${i+1}.png`, photo: `https://picsum.photos/500/300?image=${i * 5 + 10}`, date: new Date(2019,2,(Math.floor(Math.random() * 30)+1)), rating: (Math.floor(Math.random() * 6))});
```

Comment:
What is this using picsum.photos for? Is this a development placeholder or part of the finished product?